pkgs/applications/networking/p2p/jesec-rtorrent/default.nix

```diff
@@ -21,6 +21,10 @@ stdenv.mkDerivation rec {
     hash = "sha256-i7c1jSawHshj1kaXl8tdpelIKU24okeg9K5/+ht6t2k=";
   };
 
+  patches = [
+    ./avoid-stack-overflow-for-lockfile-buf.patch
```

Comment:
```suggestion
    ./000-avoid-stack-overflow-for-lockfile-buf.patch
```

Reply:
This isn't even a convention we use, though... why waste maintainer bandwidth on such a tiny change? (Also: see below, `fetchpatch` could have been used instead.)

Reply:
This isn't even a convention we use, though... why waste maintainer bandwidth on such a tiny change? (Also: see below, `fetchpatch` could have been used instead.)